### DISCUSSION TITLE
Перенести VDS на новый аккаунт

### DISCUSSION THREAD

**Discussion by vldekov** [totumonline/totum-ru-issues-and-discussions#553] [Как это сделать?] [answered]
Есть задача перенести рабочий VDS на новый аккаунт внутри Netangels. Подскажите правильный порядок переноса VDS на новый аккаунт. Также необходимо привязать этот VDS на новом аккаунте к новому домену.

**Comment by Re-Requeste** [comment]
Если брать тотум, думаю так:
1) [Импорт/экспорт ](https://ru.docs.totum.online/export#intro)
- с данными или без

2) [На всякий можно сделать бекап БД, мало ли ...](https://ru.docs.totum.online/backup)

3) [Новый домен: ](https://ru.docs.totum.online/ubuntu)  , автоустановка или ручная кому как удобнее, там же есть инструкция как ip сменить доменом (если вдруг надо)

Или почитать, есть ли возможность на ubuntu сделать образ/резв.копирование и перенести на другой идентичный/не идентичный сервер, без потерь данных

**Reply by vldekov** [reply]
![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/114401808/33bf0af5-1bfc-44d9-8cdd-1b14fd53e33b)
1) nano /home/totum/totum-mit-docker/totum/Conf.php - не находится такой путь
2) Установите Certbot и выпустите сертификат по [этой инструкции](https://ru.docs.totum.online/ubuntu#certbot). -в моем случае тоже делать?
3) где взять данные для настройки DNS (mail._domainkey)?

**Reply by Re-Requeste** [reply]
Попробуйте /home/totum/totum-mit/Conf.php 
![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/137899742/b2fa6e5b-7c5e-47b9-95f3-ec75f6ce72e8)

Про 2 пункт я не знаю как вы будете устанавливать, чистую сборку на что будете ставить ip или домен*? Можете сразу на домен, и пропустить этот пункт. Там сразу выдадут ssl сертификат и т.п.

DNS(mail.domainkey) я даже не знаю что это,и где это,  не могу подсказать. 
__В моем понимание DNS это для перенаправление домена на определенный сервер, если это и в Вашем случае, тогда Вам нужно обратиться туда где приобрели домен чтобы настроить перенаправление домена на IP адрес сервера, который вы арендовали. __ 

**P/S Забыл уточнить, у меня стоит версия PRO, возможно на mit по другому**

**Comment by vldekov** [comment, marked as answer]
Большое спасибо Всем! Есть ошибки в инструкции, Обещали исправить. Сайт перенес. Пока полет нормальный.

**Reply by ttm-alexeypolunin** [reply]
Да, доки поправим чуть позже
